src/app/modules/ix-date/ix-date.component.html

```diff
@@ -0,0 +1,6 @@
+<span
+  class="date-spanner"
+  [matTooltip]="'Machine Time\n {time}, {tz}' | translate: { time: (serverTime | formatDateTime: dateFormat: timeFormat), tz: machineTimezone }"
```

Comment:
I think we should only show tooltip when there is a mismatch between server time and browser time.